cekit-cache-server/src/main/java/org/kie/cekit/cacher/builds/yaml/YamlFilesHelper.java

```diff
@@ -29,8 +31,6 @@
      */
     public Modules load(String file) {
         ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
-        mapper.enable(SerializationFeature.INDENT_OUTPUT);
-        mapper.enable(JsonParser.Feature.ALLOW_YAML_COMMENTS);
```

Comment:
Curious why we don't want theses features enabled anymore?

Reply:
no, the comments will be removed anyway. I initially added to test but forgot to remove.

Reply:
Okee-dokee.